site/docs/v1/tech/apis/applications.adoc

```diff
@@ -55,6 +55,44 @@ include::docs/v1/tech/apis/_x-fusionauth-tenant-id-header-create-operation.adoc[
 include::docs/v1/tech/apis/_application-request-body.adoc[includeRoles=true]
 :includeRoles!:
 
+[.api-authentication]
```

Comment:
This should be consistent with our other APIs that have this same pattern, review the others.